modules/claims_api/app/models/claims_api/evss_claim.rb

```diff
@@ -5,6 +5,19 @@ class EVSSClaim
     include Virtus.model
     include ActiveModel::Serialization
 
+    EVIDENCE_GATHERING = 'Gathering of evidence'
```

Comment:
What would you think about providing a comment and link that points back to the front end code as the source for this mapping?

Reply:
I _think_ that makes sense... My question would be where do we want the source of truth to be, if we're always mirroring what the front end does I'm fine with that. If the intent is for us to become the source of truth, I would think they would reference us

Reply:
I agree! I'll add a block comment with some description

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/2701/commits/6a201d61ccbf1bec8a982f3855d9e00f166ca008

@dhall I think that encapsulates what we discussed